components/blitz/src/omero/cmd/mail/SendEmailRequestI.java

```diff
@@ -74,17 +74,13 @@ public SendEmailRequestI(MailUtil mailUtil) {
     public void init(Helper helper) {
         this.helper = helper;
 
-        final EventContext ec = ((LocalAdmin) helper.getServiceFactory()
```

Comment:
We do not have any other way to send notification when share is created. IShare cannot use SendEmailRequests within a server.